### ISSUE TITLE
cli fails with "source driver: unknown driver gcs (forgotten import?)"

### ISSUE THREAD

**Issue by dwwoelfel** [golang-migrate/migrate#52]
When I run the 3.2.0 cli version of gomigrate, I get:

```
$ ./gomigrate -source gcs://some-bucket/some-prefix -database postgres://localhost:5432/some_db up
error: source driver: unknown driver gcs (forgotten import?)
```

**Comment by dhui** [comment]
Did you build the binary yourself? It looks like you're missing the gcs source driver.

See the [CLI docs](https://github.com/golang-migrate/migrate/tree/master/cli#with-go-toolchain) about building the CLI. You'll need to specify the `google-cloud-storage` tag.

For simplicity, I'd use the pre-built binaries from the [release page](https://github.com/golang-migrate/migrate/releases) which supports most sources and DBs

**Comment by sgrove** [comment]
I'm running into this as well with both the releases from `https://github.com/golang-migrate/migrate/releases/download/v3.2.0/migrate.$platform-amd64.tar.gz` (with either `linux` or `darwin` in place of `$platform).

Here's how I'm currently downloading it:

```sh
set -x
set -e

platform='unknown'
unamestr=`uname`
if [[ "$unamestr" == 'Linux' ]]; then
   platform='linux'
elif [[ "$unamestr" == 'Darwin' ]]; then
   platform='darwin'
fi

curl -L -O https://github.com/golang-migrate/migrate/releases/download/v3.2.0/migrate.$platform-amd64.tar.gz
tar -xzvf migrate.$platform-amd64.tar.gz
mv migrate.$platform-amd64 gomigrate
rm migrate.*.tar.gz
./gomigrate -version
```

Then subsequently:

```sh
$ ./gomigrate -source gcs://some-bucket/some-prefix -database postgres://localhost:5432/some_db up
error: source driver: unknown driver gcs (forgotten import?)
```

**Comment by kewde** [comment]
https://github.com/golang-migrate/migrate/blob/master/database/sqlite3/sqlite3_test.go#L52

^shouldn't that load the right driver by the way? It's pointing to ql, I believe that should be sqlite3?

I'm also getting the error on sqlite3.
```
2018/06/13 20:23:22 source driver: unknown driver file (forgotten import?)
```

**Comment by dhui** [comment]
@kewde 
You're having a different issue since you're using `migrate` via the library instead of the CLI.

Please see the [the docs](https://github.com/golang-migrate/migrate#use-in-your-go-project) on how to use `migrate` as a library.

You're probably not importing the DB driver as the example in the docs show. If you're continuing to have problems, open a new issue to not cross threads.

**Comment by dhui** [comment]
Fixed in [v3.3.0](https://github.com/golang-migrate/migrate/releases/tag/v3.3.0)
